packages/docs/src/en/directives/teleport.md

```diff
@@ -7,7 +7,7 @@ graph_image: https://alpinejs.dev/social_teleport.jpg
 
 # Teleport Plugin
```

Comment:
# x-teleport

Reply:
Woops another one. How did I miss that? I'll do another PR as this one is already merged.